### PR TITLE
Add column resizing & footer to Page Objects in test

### PR DESCRIPTION
**Pull request by billy-addepar**

### CODE REVIEW

**Comment by billy-addepar** [comment]
@Addepar/ice

File: addon/components/ember-table.js [hunk 1]
```diff
@@ -175,8 +175,10 @@ export default class EmberTable2 extends Component {
     });
 
     this._tableResizeSensor = new ResizeSensor(this.element, () => {
```
Comment (pzuraq) [inline addon/components/ember-table.js]: This seems like it should be a separate PR/bugfix. Also, we should try to use `ember-resize-sensor` soon, that's what we built it for after all
